lucien-ios/Dashboard/DashboardViewController.swift

```diff
@@ -48,6 +66,27 @@ final class DashboardViewController: UIViewController, UIScrollViewDelegate {
         borrowingLabel.addTextSpacing(spacing: 0.5)
         myCollectionLabel.addTextSpacing(spacing: 0.7)
         comicCountLabel.text = String(viewModel.myComicsViewModel.getComicCount()) + " Comics"
+        lendingEmptyImageView.layer.cornerRadius = CGFloat(6.0)
+    }
+
+    func setLendingEmptyAddBook() {
+        lendingEmptyImageView.isUserInteractionEnabled = true
+        let lendingEmptyAddBookTapped: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(DashboardViewController.addBookButtonPressed))
+        lendingEmptyAddBookTapped.numberOfTapsRequired = 1
+        lendingEmptyImageView.addGestureRecognizer(lendingEmptyAddBookTapped)
+    }
+
+    func setPlaceholders() {
+        if viewModel.borrowingViewModel.getComicCount() == 0 {
+            borrowingEmptyImageView.isHidden = false
+            borrowingEmptyDescriptionLabel.isHidden = false
+        }
+        if viewModel.lendingViewModel.getComicCount() == 0 {
+            lendingEmptyImageView.isHidden = false
+            lendingEmptyDescriptionLabel.isHidden = false
+            lendingEmptyPlusImageView.isHidden = false
+            lendingEmptyAddBookLabel.isHidden = false
```

Comment:
Looks like the add book UI should only come up if the user has no comics in their collection - so a different one should come up here